content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -57,14 +57,18 @@ The following operating systems are officially supported by `dd-trace`. Any oper
 
 | dd-trace Version    | Operating System      | Architectures         | Minimum Versions                         |
 | ------------------- | --------------------- | --------------------- | ---------------------------------------- |
+| 5.x                 | Linux (glibc)         | arm64, x64       | CentOS 7, Debian 9, RHEL 7, Ubuntu 14.04 |
```

Comment:
Pending to confirm this Minimum Versions and apply the changes in `content/en/tracing/trace_collection/compatibility/nodejs.md`